bin/npmgraph.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+const path = require('path');
+const express = require('express');
+const open = require('open');
+const argv = require('minimist')(process.argv.slice(2));
+
+const app = express();
+const port = 3003;
+let packageName = '';
+
+const getPackageName = (packlist = []) => {
+  if (packlist.length > 0) return packlist[0];
+  console.log('Please assign packageName');
```

Comment:
It's weird that this is logged here, but that the actual action related to this (exiting the program) happens elsewhere.  See comment below.